sympy/polys/matrices/ddm.py

```diff
@@ -92,20 +92,32 @@ def __init__(self, rowslist, shape, domain):
         if not (len(self) == m and all(len(row) == n for row in self)):
             raise DDMBadInputError("Inconsistent row-list/shape")
 
+    def getitem(self, key):
+        i, j = key
+        m, n = self.shape
+
+        if i >= m:
+            raise IndexError("row index out of range")
+        if j >= n:
+            raise IndexError("column index out of range")
+
+        return self[i][j]
```

Comment:
Since `DomainMatrix.__getitem__` has `i` and `j` separately and then puts them into a tuple just to be unpacked into `i` and `j` here I think it would be better to change the signature of `getitem` to take `i` and `j` as separate arguments. Also I don't think we need to check the row/column indices because `self[i][j]` will raise if needed. I think this can just be:
```
def getitem(self, i, j):
    return self[i][j]
```

Reply:
Sorry I missed this.
Thanks.